packages/bsky/src/data-plane/server/routes/activity-subscription.ts

```diff
@@ -0,0 +1,28 @@
+import { ServiceImpl } from '@connectrpc/connect'
+import { Service } from '../../../proto/bsky_connect'
+import { Database } from '../db'
+import { IsoSortAtKey } from '../db/pagination'
+
+export default (db: Database): Partial<ServiceImpl<typeof Service>> => ({
+  async getActivitySubscriptions(req) {
+    const { actorDid, cursor, limit } = req
+
+    let builder = db.db
+      .selectFrom('activity_subscription')
+      .select('subjectDid')
+      .where('creator', '=', actorDid)
+      .orderBy('indexedAt', 'desc')
```

Comment:
Perhaps it's not necessary, but I didn't see an index supporting this pagination scheme.